addons/mail/static/src/js/views/activity/activity_controller.js

```diff
@@ -66,21 +66,30 @@ var ActivityController = BasicController.extend({
      */
     _onScheduleActivity: function () {
         var state = this.model.get(this.handle);
-        Component.env.services.dialog.add(SelectCreateDialog, {
-            resModel: state.model,
-            searchViewId: this.searchViewId,
-            domain: this.model.originalDomain,
-            title: sprintf(_t("Search: %s"), this.title),
-            noCreate: !this.activeActions.create,
-            multiSelect: false,
-            context: state.context,
-            onSelected: async resIds => {
-                const messaging = await owl.Component.env.services.messaging.get();
-                const thread = messaging.models['Thread'].insert({ id: resIds[0], model: this.model.modelName });
-                await messaging.openActivityForm({ thread });
-                this.trigger_up('reload');
+        Component.env.services.dialog.add(
+            SelectCreateDialog,
+            {
+                resModel: state.model,
+                searchViewId: this.searchViewId,
+                domain: this.model.originalDomain,
+                title: sprintf(_t("Search: %s"), this.title),
+                noCreate: !this.activeActions.create,
+                multiSelect: false,
+                context: state.context,
+                onSelected: async (resIds) => {
+                    const messaging = await owl.Component.env.services.messaging.get();
+                    const thread = messaging.models["Thread"].insert({
+                        id: resIds[0],
+                        model: this.model.modelName,
+                    });
+                    await messaging.openActivityForm({ thread });
+                    this.trigger_up("reload");
```

Comment:
```suggestion
```
Having the reload here as well produces 2 reloads when the activity is created via the FormViewDialog (when a row is clicked).

It seems to work fine, but don't hesitate to test a bit